<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DS_Create</class>
 <widget class="QWidget" name="DS_Create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dataset Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lE_dataset_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Dataset Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlo_dstype">
     <item>
      <widget class="QRadioButton" name="rB_dstype_image">
       <property name="text">
        <string>Image</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_dstype_Statistics">
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dataset Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pTE_ds_desc"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlo_ds_bt">
     <item>
      <widget class="QPushButton" name="pB_ds_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_ds_create">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
